lib/math.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var R = require('ramda');
+
+var operators = {
+  '+': R.add,
+  '-': R.subtract,
+  '*': R.multiply,
+  '/': R.divide,
+  '%': R.modulo,
```

Comment:
I did, but when it said it didn't work like the JavaScript `%` does, I wasn't sure if that would be expected behavior or not.

I still don't understand what it does, so I'd happily yield to someone else's judgement in this case. 😄